<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="char_createNewChar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文细黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>创建新玩家</string>
   </property>
   <widget class="QLabel" name="char_name_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>31</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>角色名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="char_PLid_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>71</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>玩家名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="char_PLid_input">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>71</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="char_qnum_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>71</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="char_identity_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>31</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>玩家身份：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="char_identity_select">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>31</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>KP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>骰子</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NPC</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="char_name_input">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>31</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="char_qnum_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>71</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QQ号/邮箱：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="pic_autochange">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="pic_keyword_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>453</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>差分</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>关键词</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pic_get_fps_btn">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>43</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>读取立绘差分列表</string>
     </property>
    </widget>
    <widget class="QToolButton" name="char_imgdir_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>64</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="char_imgdir_label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文细黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>角色立绘路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="char_imgdir_input">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="pic_default_input">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="pic_default_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文细黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>模板内默认图：</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="pic_enable">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>104</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>启动立绘自动替换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="char_save_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>524</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="char_cancel_btn">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>524</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
